hooks/migrate_csi_endpoint.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
+#
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /deckhouse/shell_lib.sh
+
+get_config() {
+  echo '{"configVersion":"v1", "beforeHelm": 10}'
+}
+
+run_trigger() {
+  export old_driver_name="linstor.csi.linbit.com"
+  export new_driver_name="replicated.csi.storage.deckhouse.io"
+  export old_attacher="linstor-csi-linbit-com"
+  export new_attacher="replicated-csi-storage-deckhouse-io"
+  export LABEL_KEY="storage.deckhouse.io/need-kubelet-restart"
+  export LABEL_VALUE=""
+  export SECRET_NAME="csi-migration-finished"
```

Comment:
```suggestion
  export SECRET_NAME="csi-migration-completed"
```
Completed is better, and there is value `internal.csiMigrationHook.completed`.